artifact/gem5art/artifact/_filedb.py

```diff
@@ -0,0 +1,90 @@
+import json
+from pathlib import Path
+from typing import Optional, Iterable, List, Dict, Union, Tuple, Any
+from uuid import UUID
+import copy
+
+class FileDB:
+
+    """
+        An artifact database with a json file backend.
+    """
+
+    __json_file: Path
+    __uuid_artifact_map: Dict[UUID, Dict[str,str]]
+    __hash_uuid_map: Dict[str, List[UUID]]
```

Comment:
Shouldn't this be 1-to-1? Why a list?

Reply:
I did that just in case. md5 has known collisions.